rllib/utils/pre_checks/env.py

```diff
@@ -209,6 +224,7 @@ def get_type(var):
         if not env.observation_space.contains(temp_sampled_next_obs):
             raise ValueError(error)
     _check_done(done)
+    _check_done(truncated)
```

Comment:
Need to add flag to this function to change the error message to be about truncated or done.